sql/telemetry_derived/clients_first_seen_v1/query.sql

```diff
@@ -42,6 +42,12 @@ SELECT
       NULL
     WHEN
       previous.second_seen_date IS NULL
+      AND today.client_id IS NULL
+    THEN
+      NULL
+    WHEN
+      previous.second_seen_date IS NULL
+      AND today.client_id IS NOT NULL
```

Comment:
This is fine to merge as-is, but I think it would be easier to understand what this is doing as an IF statement:
```sql
IF(
  previous.first_seen_date IS NOT NULL
  AND previous.second_seen_date IS NULL
  AND today.client_id IS NOT NULL,
  @submission_date,
  previous.second_seen_date
) AS second_seen_date
```

Reply:
I also updated the first_seen_date computation to be an IF statement for consistency.